### DISCUSSION TITLE
Feature suggestion (Swap customer name with customer mobile as a unique ID)

### DISCUSSION THREAD

**Discussion by magedfathi** [Blair2004/NexoPOS#554] [Ideas]
in my case i am using the customer mobile number as a customer name cause for large data we will find duplication for the same name like ahmed or mohamed even the second name might duplicate but mobile number cant duplicate and the correct way is to select the customer first >> search by mobile number >> if not existing, we will create new customer with mobile number as a customer name but if by mistake i create 2 customer with same mobile the system will accept in my case so i suggest to swap the customer name with mobile number from the system itself to avoid the duplication and we will save the name as an optional for who like because for large operation the most important is the mobile number and email for marketing.

![customer duplicate](https://user-images.githubusercontent.com/87772530/152676147-3cdb0105-127b-41b0-a2dc-6301940564e9.png)

**Comment by Blair2004** [comment]
Hi,
Thank you for this idea.

Unfortunately we cannot swap the name to the number. However, we might add a feature that force the phone number to be unique (option to toggle).

How about that ?

**Comment by magedfathi** [comment]
totally agree. it will achieve the purpose
